src/mscorlib/shared/System/Span.Fast.cs

```diff
@@ -154,6 +154,13 @@ internal Span(ref T ptr, int length)
 #endif
         }
 
+        /// <summary>
+        /// Returns a reference to the 0th element of the Span. If the Span is empty, returns null reference.
+        /// It can be used for pinning and is required to support the use of span within a fixed statement.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public unsafe ref T GetPinnableReference() => ref (_length != 0) ? ref _pointer.Value : ref Unsafe.AsRef<T>(null);
```

Comment:
Rebase?

Reply:
bleh

Reply:
Had rebased to do asm diffs against master; forgot so it made the sync bad